src/main/java/orca/flukes/ndl/ManifestLoader.java

```diff
@@ -388,85 +387,17 @@ public void ndlLinkConnection(Resource l, OntModel m,
 			setRequestGuid(l, ml);
 			
 			nodes.put(getTrueName(l), ml);
-			
-			// special case handling - if storage is one side of the link
-			// (only for shared vlan storage), then we only remember
-			// interfaces with no ip addresses on them (others are duplicates; 
-			// we save the equivalence relationship to lookup IP later) /ib 09/18/14
-			boolean sharedVlanStorageLink = false;
-			Map<Resource, List<Resource>> interfacesByOwner = new HashMap<Resource, List<Resource>>();
-			
-			for(Resource ti: interfaces) {
-				List<Resource> attached = NdlCommons.getWhoHasInterface(ti, m);
-				if (attached == null)
-					continue;
-				for(Resource ta: attached) {
-					// skip links (self included)
-					if (NdlCommons.hasResourceType(ta, NdlCommons.topologyLinkConnectionClass))
-						continue;
-					if (NdlCommons.isISCSINetworkStorage(ta)) {
-						sharedVlanStorageLink = true;
-					}
-					List<Resource> tmp = interfacesByOwner.get(ta);
-					if (tmp == null)
-						tmp = new ArrayList<Resource>();
-					tmp.add(ti);
-					interfacesByOwner.put(ta, tmp);
-				}
-			}
-
-			// map ip/noip interfaces to each other
-			for(Map.Entry<Resource, List<Resource>> ee: interfacesByOwner.entrySet()) {
```

Comment:
7. Since the loop above fills out interfacesByOwner map, which is no longer used, we don't need this loop either. Notice sharedStorageLinkInterfaceEquivalence map in line 430 and go to item 8